### PR TITLE
Persist all events

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: test/mocks/batch-event.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const event = require('./event')
+
+const FILENAME = 'batch-file.dat'
```
Comment (simonnineteen90) [inline test/mocks/batch-event.js:3]: should this be split out into a const?

File: test/mocks/event.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
```
Comment (simonnineteen90) [inline test/mocks/event.js:1]: is it worthwhile to split some of these into consts?

File: app/inbound/save-event/get-warning-type.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const WARNING_IDENTIFIER = '.warning.'
```
[the remark this answers is not in the document]
Comment (johnwatson484) [reply app/inbound/save-event/get-warning-type.js]: Yep, updated.

File: test/mocks/batch-event.js (hunk 1 above)
Comment (johnwatson484) [reply test/mocks/batch-event.js:3]: I'm not sure it needs to be, it's not used anywhere else but this file.

I could add a mock filename module, but feels like overkill.

Maybe if it was used across the tests that would be valuable.

File: test/mocks/event.js (hunk 2 above)
Comment (johnwatson484) [reply test/mocks/event.js]: You've convinced me.  I've split the mocks folder into `events` and `values`.  Where events objects are made up of mock values.  Given the nature of this service, I don't think will get too complicated as the need to have different values in tests is limited.

File: test/mocks/batch-event.js (hunk 1 above)
Comment (johnwatson484) [reply test/mocks/batch-event.js]: Change my mind to be consistent with other extension of mocks in tests

**Review by simonnineteen90 — APPROVED** [review]

**Pull request merged**